rebar.config

```diff
@@ -33,8 +31,6 @@
 
 {post_hooks,[]}.
 
-{erl_first_files, ["src/emqx_logger.erl", "src/emqx_rule_actions_trans.erl"]}.
```

Comment:
do we need to add `emqx` as a dependent of other apps to make sure compiling `emqx` app first?

Reply:
rebar3 should detect this dependency.
for some reason this did not work when src was in root dir.